wild/tests/sources/undefined_symbols.cc

```diff
@@ -0,0 +1,16 @@
+
+//#AbstractConfig:default
+//#DiffEnabled:false
+//#RunEnabled:false
+
+//#Config:executable:default
+//#LinkArgs:--cc=g++ -fPIC -static
```

Comment:
Tests that link against libc are moderately slow, so we do need to be a little bit conservative with adding new ones. An alternative to adding this would be to adjust one of the existing tests to trigger the bug. One way to do this would be to add the flag `-fPIC` to `libc-integration.c` in the config `clang-static-pie`.

Reply:
It works but seems kinda fragile. I doubt I'd be able to come up with something better, though.